src/components/header/NavLink.tsx

```diff
@@ -0,0 +1,24 @@
+import { NavLink } from "../../types/interfaces";
+
+export const navData: NavLink[] = [
+  {
+    text: "Basic Resources",
+    link: "/basic-resources",
+    title: "",
+  },
+  {
+    text: "Advanced Resources",
+    link: "/",
+    title: "Select Level of Co-Design Practice",
+  },
+  {
+    text: "Tools Resources",
+    link: "/tools-resources",
+    title: "",
+  },
+  {
+    text: "Case Studies & CORE Toolkits",
+    link: "/case-studies-and-core-tools",
+    title: "Level of Co-Design Practice",
+  },
+];
```

Comment:
Can you put this file in utils folder and give it .ts extension as it is not a tsx file

Reply:
fixed